recipes-devtools/python/python3-janus_1.0.0.bb

```diff
@@ -3,8 +3,8 @@ HOMEPAGE = "https://pypi.org/project/janus/"
 LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=23878c357ebb4c8ce1109be365043349"
 
-SRC_URI[sha256sum] = "4712e0ef75711fe5947c2db855bc96221a9a03641b52e5ae8e25c2b705dd1d0c"
+SRC_URI[sha256sum] = "df976f2cdcfb034b147a2d51edfc34ff6bfb12d4e2643d3ad0e10de058cb1612"
 
-inherit pypi setuptools3
+inherit python_setuptools_build_meta pypi
```

Comment:
Maybe give a short hint about the adapted `inherit` in the commit message

Reply:
Done.